### ISSUE TITLE
what is the validation set?

### ISSUE THREAD

**Issue by cpwan** [Yuxiang1995/ICDAR2021_MFD#5]
In [formula_detection.py](https://github.com/Yuxiang1995/ICDAR2021_MFD/blob/main/configs/_base_/datasets/formula_detection.py), the annotation file is under Ts01, but the images are under Tr01. Also, the training set contains Va00, Va01. Shouldn't they belong to the validation set?

```    
    val=dict(
        type=dataset_type,
        ann_file=data_root + 'Ts01/train_coco_sdk4.json',
        img_prefix=data_root + 'Tr01/img/',
        classes=classes,
        pipeline=test_pipeline),
    test=dict(
        type=dataset_type,
        ann_file=data_root + 'Ts10/train_coco_sdk4.json',
        img_prefix=data_root + 'Tr10/img/',
        classes=classes,
        pipeline=test_pipeline))
```

**Comment by Yuxiang1995** [comment]
Hi @cpwan , I'm sorry for my carelessness. Images must be as same as their annotations, 'Tr' should change to 'Ts' here, and the latest code has been fixed this error. 
This config is my final version which gets the result of the unlabeled data Ts10. If you only want to reimplement the result on [README.md](https://github.com/Yuxiang1995/ICDAR2021_MFD/blob/main/README.md) quickly, train on Tr00, Tr01, Va00 and Va01, validate on Ts00, and test on Ts01.
